spec/classes/candlepin_spec.rb

```diff
@@ -36,9 +36,11 @@
             'log4j.logger.org.hibernate.internal.SessionImpl=ERROR',
             'pinsetter.org.candlepin.pinsetter.tasks.ExpiredPoolsJob.schedule=0 0 0 * * ?',
             'pinsetter.org.candlepin.pinsetter.tasks.CertificateRevocationListTask.schedule=0 0 0 1 1 ?',
+            'candlepin.audit.hornetq.config_path=/etc/candlepin/broker.xml',
           ])
         end
         it { is_expected.to contain_file('/etc/tomcat/tomcat.conf') }
+        it { is_expected.to contain_file('/etc/candlepin/broker.xml') }
```

Comment:
In general I like to verify lines that are templated to ensure it's correct. Something like this:

```suggestion
        it { is_expected.to contain_file('/etc/candlepin/broker.xml').with_content(%r{<acceptor name="stomp">tcp://localhost:61613?protocols=STOMP;sslEnabled=true;trustStorePath=$PATH;trustStorePassword=$PASSWORD;keyStorePath=$PATH;keyStorePassword=$PASSWORD;needClientAuth=true</acceptor>}) }
```

Reply:
When I look at the generated template from the test I get:

```
       +            <acceptor name="stomp">tcp://:?protocols=STOMP;sslEnabled=true;trustStorePath=;trustStorePassword=;keyStorePath=;keyStorePassword=;needClientAuth=true</acceptor>
```

This doesn't appear to include the actual parameters being filled in. Is this because I am using `@` syntax vs `scope` syntax in the template?

Reply:
Yes, that's probably the issue. They're not in the local scope of that class. I've been considering moving to [EPP](https://puppet.com/docs/puppet/latest/lang_template_epp.html) since it's more explicit about scope. This is probably also why it's not actually binding on port 61613 in the acceptance tests and the tests are red.